lib/src/utils/enums/syntax_language.dart

```diff
@@ -0,0 +1,23 @@
+enum SyntaxLanguage {
```

Comment:
In order to have one source of true, client of library should get languages and themes from plugin. We do not want to have these enum classes